resources/uk/gov/hmcts/npm/check-deprecated-npm-dependency.sh

```diff
@@ -6,7 +6,7 @@ REQUIRED_VERSION=${2}
 
 # Attempt to find angular version in a package.json file and trims version of angular/core to get the major version value 
 CURRENT_VERSION=""
-for file in $(find . -type f -name "package.json" -not -path "./node_modules/*"); do
+for file in $(find . -type f -name "package.json" -not -path "*/node_modules/*" -not -path "*/dist/*"); do
```

Comment:
why does it need to check outside of the package.json at the root? is there any projects where the frontend isn't in the root package.json?

Reply:
and why can't you use the CLI, to get the version instead of all that sed, tr and cut =/

https://stackoverflow.com/a/70254297

Reply:
em-showcase flagged this in DTSPO-17287

Reply:
I could get the angular cli version, but struggled to get the angular core version

Reply:
I also hadn't used yarn info as I couldn't tell if that would be suitable for every single repo, in case some didn't use yarn

Reply:
I've added it in https://github.com/hmcts/cnp-jenkins-library/pull/1222/commits/5de0de3341b0695d6cd69627bc2a1a7b0bc66e07 -- will re-run against a few apps I know are using angular too

Reply:
@timja  example of this running too with those suggested yarn changes: https://build.hmcts.net/job/HMCTS_d_to_i/job/em-showcase/job/PR-498/6/consoleFull

Reply:
> DTSPO-17287

aware but why is it checking anything outside of the root package.json

> I also hadn't used yarn info as I couldn't tell if that would be suitable for every single repo, in case some didn't use yarn

Its a pipeline requirement to use yarn, any Jenkins pipeline repository will be using it.

>  that also returns a few different angular/core versions which are child dependencies too

I'd expect it to be one at the root level but would need testing.

Reply:
` pipeline requirement to use yarn` - Wasn't aware of this sorry, I do see now there's a small bit in the readme though

Are you able to re-review please @timja ? Pushed some changes to use yarn now and it is only checking the root level